package.json

```diff
@@ -216,7 +211,7 @@
 		"electron-is-dev": "^2.0.0",
 		"electron-store": "^8.1.0",
 		"electron-updater": "^6.1.4",
-		"is-online": "^9.0.1",
+		"is-online": "^10.0.0",
```

Comment:
This one cannot be updated due to the ESM way of import.
By the way, that's the only change between v9 and v10 as far as I remember.